source/cellular_r4_urc_handler.c

```diff
@@ -179,7 +179,8 @@ static CellularPktStatus_t _parseUrcIndicationCsq( CellularContext_t * pContext,
 
     if( atCoreStatus == CELLULAR_AT_SUCCESS )
     {
-        if( ( retStrtoi >= INT16_MIN ) && ( retStrtoi <= ( int32_t ) INT16_MAX ) )
+        /* Possible value is 0 to 5. */
+        if( ( retStrtoi >= 0 ) && ( retStrtoi <= 5 ) )

```

Comment:
Could you help share the reason that possible balue is only 0 to 5? And also update that to the comment.

Reply:
Update with more description. This is the bar level returned from cellular modem. Possible value is 0 to 5.